lib/jobs/dell-wsman-inventory.js

```diff
@@ -158,11 +158,13 @@ function DellWsmanInventoryJobFactory(BaseJob, Logger, Promise, util, waterline,
       	var requestUri = '';
       	if(self.nodeType === 'compute'){
       		requestUri = self.dellConfigs.services.inventory.serverCallback;
+            if(type === 'manager'){  // special case for manager endpoint - use 2.0 api
```

Comment:
Special case to use new 2.0 api in the inventory service.  This can be removed if we shift all wsman inventory collection to the 2.0 api.  The 2.0 api will provide all data returned by iDRAC without translation to model.